portfolio/src/main/webapp/script.js

```diff
@@ -119,3 +119,33 @@ function hideOnMouseout(hoveredItem) {
     tag.style.filter = 'brightness(100%)';
   }
 }
+
+/**
+ * Fetches the message from the server /data and adds it to the DOM.
+ */
+function getMessage() {
+  const responsePromise = fetch('/data');
+  
+  responsePromise.then(handleResponse);
+}
+
+/**
+ * Handles response by converting it to text and passing the result
```

Comment:
I would rephrase this comment because now the user needs to look up what addMessageToDom does to understand what this function does. I would re-write it as: Obtains the reponse's text and adds it to the DOM.

Reply:
I believe this method only converts the response to text. The next method (addMessageToDom()) would add it to the DOM instead. Is it okay if I re-write it to say "Obtains the response's text" instead?